test/test-start.js

```diff
@@ -0,0 +1,37 @@
+var assert = require("assert"); // assert module is supported from nodejs.
+const execSync = require('child_process').execSync;
+
+describe('Module check', function(){

```

Comment:
Please do lint-check for this file.
This file doesn't follow our coding rule overall.